front-end/src/components/player-page/stats/win-rate-box.tsx

```diff
@@ -0,0 +1,53 @@
+import RisenBox1 from "../../risen-box/risen-box-1";
+import {Cell, Label, Pie, PieChart, ResponsiveContainer, Tooltip} from "recharts";
+import React from "react";
+import {useTheme} from "@emotion/react";
+import {Theme, Typography} from "@mui/material";
+import "./win-rate-box.css"
+
+interface WinRateBoxProps {
+    wins: number;
+    losses: number;
+    hasData: boolean;
+}
+
+export default function WinRateBox(winRateProps: WinRateBoxProps) {
+    const theme = useTheme() as Theme;
+
+    // Fix this later lol
+    const COLORS = [theme.palette.primary.dark, theme.palette.secondary.dark];
+
+    const data = [{name: 'Wins', value: winRateProps.wins}, {name: 'Losses', value: winRateProps.losses}]
+    const winRate = Math.round((winRateProps.wins / (winRateProps.wins + winRateProps.losses)) * 100)
```

Comment:
Hm. Can we create a util for this instead? As I think I will use it in the future as well. Actually I alsoo have one you could use. `calculateWR`.